app/modules/import-export/utils.php

```diff
@@ -81,7 +81,8 @@ public static function get_registered_cpt_names() {
 		unset(
 			$post_types[ Landing_Pages_Module::CPT ],
 			$post_types[ Source_Local::CPT ],
-			$post_types[ Conversion_Center_Module::CPT_LINKS_PAGES ]
+			$post_types[ Conversion_Center_Module::CPT_LINKS_PAGES ],
```

Comment:
@nicoladj77 Will there be a separate PR to remove CPT_LINKS_PAGES?

Reply:
@louiswol94 yes I have it ready when this is merged.